Project/cypress/e2e/claim.spec.cy.ts

```diff
@@ -0,0 +1,85 @@
+
+import { id, submitClaim } from "../support/helper/claim";
+import { addEvent, deleteEvent } from "../support/helper/event";
+import { addExpense, deleteExpense } from "../support/helper/expense";
+import { AssignClaim } from "../support/pageobjects/AssignClaim";
+import { Employee } from "../support/pageobjects/Employee";
+
+
+let empNumber: string;
+let eventName = "EventART";
+let eventId: string;
+let expenseName = "ExpenseART";
+let expenseId: string;
+let amount = "5";
+describe("", () => {
+    beforeEach(function () {
+        cy.visit("/web/index.php/auth/login");
```

Comment:
move it to the command itself please, and it should have intercepts and dynamic wait